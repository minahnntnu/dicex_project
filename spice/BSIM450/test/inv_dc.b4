** MOSFET Inverter (DC): Benchmarking Implementation of BSIM4 by Jane Xi 05/09/2003.

*** circuit description ***
m10 out in 0 0 n1 L=0.18u W=10.0u rgeoMod=1
m20 out in d d p1 L=0.18u W=10.0u rgeoMod=1
Vd d 0 dc 2.0
vin in 0 0.0
.dc vin 0 2.0 0.05

*.tran 0 20ns 0.01ns

.option reltol=1e-5
.print dc v(in) v(out)

.include modelcard.nmos
.include modelcard.pmos
.end





