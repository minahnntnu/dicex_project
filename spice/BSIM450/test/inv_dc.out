Note: can't find init file.

Circuit: ** MOSFET Inverter (DC): Benchmarking Implementation of BSIM4 by Jane Xi 05/09/2003.

Circuit: ** MOSFET Inverter (DC): Benchmarking Implementation of BSIM4 by Jane Xi 05/09/2003.
Date: Fri Jul 29 12:44:09  2005


** MOSFET Inverter (DC): Benchmarking Implementation of BSIM4 by Jane Xi 05/09/2003.
DC transfer characteristic  Fri Jul 29 12:44:09  2005
--------------------------------------------------------------------------------
Index   sweep           v(in)           v(out)          
--------------------------------------------------------------------------------
0	0.000000e+00	0.000000e+00	1.991253e+00	
1	5.000000e-02	5.000000e-02	1.994247e+00	
2	1.000000e-01	1.000000e-01	1.996195e+00	
3	1.500000e-01	1.500000e-01	1.997031e+00	
4	2.000000e-01	2.000000e-01	1.996256e+00	
5	2.500000e-01	2.500000e-01	1.992651e+00	
6	3.000000e-01	3.000000e-01	1.984014e+00	
7	3.500000e-01	3.500000e-01	1.967376e+00	
8	4.000000e-01	4.000000e-01	1.940081e+00	
9	4.500000e-01	4.500000e-01	1.900569e+00	
10	5.000000e-01	5.000000e-01	1.847640e+00	
11	5.500000e-01	5.500000e-01	1.779023e+00	
12	6.000000e-01	6.000000e-01	1.689181e+00	
13	6.500000e-01	6.500000e-01	1.561547e+00	
14	7.000000e-01	7.000000e-01	1.283529e+00	
15	7.500000e-01	7.500000e-01	5.446047e-01	
16	8.000000e-01	8.000000e-01	2.454797e-01	
17	8.500000e-01	8.500000e-01	1.877303e-01	
18	9.000000e-01	9.000000e-01	1.546973e-01	
19	9.500000e-01	9.500000e-01	1.311166e-01	
20	1.000000e+00	1.000000e+00	1.125806e-01	
21	1.050000e+00	1.050000e+00	9.716826e-02	
22	1.100000e+00	1.100000e+00	8.386920e-02	
23	1.150000e+00	1.150000e+00	7.209782e-02	
24	1.200000e+00	1.200000e+00	6.149531e-02	
25	1.250000e+00	1.250000e+00	5.184355e-02	
26	1.300000e+00	1.300000e+00	4.300702e-02	
27	1.350000e+00	1.350000e+00	3.491480e-02	
28	1.400000e+00	1.400000e+00	2.754538e-02	
29	1.450000e+00	1.450000e+00	2.092351e-02	
30	1.500000e+00	1.500000e+00	1.512440e-02	
31	1.550000e+00	1.550000e+00	1.027761e-02	
32	1.600000e+00	1.600000e+00	6.541441e-03	
33	1.650000e+00	1.650000e+00	4.019907e-03	
34	1.700000e+00	1.700000e+00	2.640329e-03	
35	1.750000e+00	1.750000e+00	2.150244e-03	
36	1.800000e+00	1.800000e+00	2.272243e-03	
37	1.850000e+00	1.850000e+00	2.830708e-03	
38	1.900000e+00	1.900000e+00	3.766163e-03	
39	1.950000e+00	1.950000e+00	5.104705e-03	

CPU time since last call: 0.240 seconds.

Total CPU time: 0.240 seconds.

Current dynamic memory usage = 360448,
Dynamic memory limit = -18809392.

